src/lib/y2s390/clients/inst_devices_preconf.rb

```diff
@@ -0,0 +1,95 @@
+# encoding: utf-8
+
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2s390/dialogs/devices_preconf"
+
+Yast.import "Kernel"
+Yast.import "Arch"
+Yast.import "GetInstArgs"
+
+module Y2S390
+  module Clients
+    # I/O device auto-configuration is a mechanism by which users can specify IDs
+    # and settings of I/O devices that should be automatically enabled.
+    #
+    # This client allows to trigger such mechanism.
+    class InstDevicesPreconf
+      include Yast::Logger
+
+      AUTOCONFIG_FILE = "/sys/firmware/sclp_sd/config/data".freeze
+
+      # @return [Symbol] :auto, :next, :back or :abort
+      def run
+        # :auto forwards the former action (:next or :back)
+        return :auto if skip?
+
+        log.info("BEGIN of inst_devices_preconf")
```

Comment:
I think we do not need to log the `BEGIN` and `END` markers, the workflow manager already logs when it starts a client and when it finishes. If it is really needed then I'd log it as `log.debug`.

(The y2log is already too long, we should add the new logging carefully.)